tests/Core/Controller/ClientFlowLoginControllerTest.php

```diff
@@ -481,7 +490,7 @@ public function testGeneratePasswordWithPasswordForOauthClient() {
 		$this->eventDispatcher->expects($this->once())
 			->method('dispatch');
 
-		$expected = new Http\RedirectResponse('https://example.com/redirect.php?state=MyOauthState&code=MyAccessCode');
+		$expected = new Http\RedirectResponse($redirectUri);
```

Comment:
```suggestion
		$expected = new Http\RedirectResponse($redirectUrl);
```

Than the tests will pass ;)

Reply:
Whoops, sorry!